### ISSUE TITLE
Feature request: Add completions

### ISSUE THREAD

**Issue by b0o** [NullSense/fuzzy-sys#10]
Completions would be a very nice addition to this plugin! 

Alternatively, if fuzzy-sys arguments were made to be compatible with systemctl (e.g. `start` rather than `--start`), you could piggyback on the existing systemctl completions with `compdef fuzzy-sys=systemctl`

**Comment by NullSense** [comment]
Good idea! Feel free to make a pull request, if not, I will try to put some work towards this in a few weeks :slightly_smiling_face:.
